### PR TITLE
Small bugfix and several new features for the geometry comparison tool

### PR DESCRIPTION
**Pull request by cschomak**
- Fixed bug in absolute position that caused the absolute positions in the GC tool to be slightly off (global position record applied twice, 0.1-0.2 cm effect in 2016 data), relative shifts were not affected

- Changed x-axis of the displayed geometry (previously from first "new" alignment, now from the old one)

- Added profile plots to get a better feeling for the trends than from the scatter plots alone

- Added tables for the mean shifts and absolute positions of the subdetectors

- Replaced cout by LogInfo statements

### CODE REVIEW

**Comment by franzoni** [comment]
please test

File: Alignment/OfflineValidation/scripts/GeometryComparisonPlotter.cc [hunk 1]
```diff
@@ -266,7 +288,18 @@ void GeometryComparisonPlotter::MakePlots (vector<TString> x, // axes to combine
                                    _window_width,
                                    _window_height);
         c_global[ic]->Divide(x.size(),y.size());
+        
+        if (_make_profile_plots == "true") {
```
Comment (ghellwig) [inline Alignment/OfflineValidation/scripts/GeometryComparisonPlotter.cc]: @cschomak is there a reason why a boolean is represented as a string?

**Review by ghellwig — COMMENTED** [review]
fine with the changes, but unless there is no strong reason, please make the booleans emulated with TStrings real booleans

File: Alignment/OfflineValidation/scripts/GeometryComparisonPlotter.cc (hunk 1 above)
Comment (cschomak) [inline Alignment/OfflineValidation/scripts/GeometryComparisonPlotter.cc]: Not really. There were already some booleans represented as string and I did not think about it and stuck to it. I changed them now

**Comment by ghellwig** [comment]
please test

**Comment by ghellwig** [comment]
+1
provided jenkins agrees
thanks @cschomak

**Comment by davidlange6** [comment]
+1

**Pull request merged**